web/js/components/ReadSupers.jsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import Superlative from './Superlative.jsx'
+import Notifications, {notify} from 'react-notify-toast';
+
+// Hard-coding supers for testing 
+var supers = [
+  {
+  "name":"Most likely to die in a freak accident",
+  "created_by":"A",
+  "assigned_by":"B",
+  "receipient":"C"
+  },
+  {
+  "name":"Most loyal",
+  "created_by":"D",
+  "assigned_by":"E",
+  "receipient":"F"
+  },
+  {
+  "name":"Best hair",
+  "created_by":"D",
+  "assigned_by":"E",
+  "receipient":"F"
+  },
+  {
+  "name":"Most likely to become president",
+  "created_by":"D",
+  "assigned_by":"E",
+  "receipient":"F"
+  }
+]
+
+export default class ReadSupers extends React.Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      currentSuper:"",
+      supersList:[]
+    }
+
+    this.nextButtonClicked = this.nextButtonClicked.bind(this)
```

Comment:
I'm not that familiar with js, but wouldn't this do nothing?

Reply:
i can't remember exactly but this was something around fixing issues with scope whenever you call "this" inside the nextButtonClicked function. Will properly investigate and understand and update accordingly

Reply:
So this is needed and explained here (something to do with React not auto-binding for ES6): https://medium.freecodecamp.org/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56 - I may need your help understanding this concept.

Reply:
Ah okay, #3 is what I'm used to seeing.

I prefer it from a readability perspective, but I'm not picky and avoid javascript like the plague it is.  /shrug